content/blogposts/2022/release-3.41.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Sourcegraph 3.41 release'
+publishDate: 2022-06-21T10:00-07:00
+description: 'Sourcegraph 3.41 introduces running batch changes server-side, file mounting for the batch change container, greater pattern tracking in Code Insights, and admin settings to restrict extensions to those authored by Sourcegraph.'
+tags: [blog, release]
+slug: 'release/3.41'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+changelogItems: 
+  - description: 'A new templating variable, batch_change_link has been added for more control over where the "Created by Sourcegraph batch change ..." message appears in the published changeset description.'
```

Comment:
@malomarrec @lcolston08 @iskyOS - Can we re-write this in active voice: "Added a new templating variable, batch_change_link, to give you control over where the...." And then add a sentence on what the value is/why someone would do this?

Reply:
```suggestion
  - description: 'Added a new templating variable, batch_change_link, to give you more control over where the "Created by Sourcegraph batch change ..." message appears in the published changeset description.'
```

Reply:
I understand that this allows you to customize the changeset description so that you don't need to have the Sourcegraph backlink always at the end of the message. @malomarrec are there specific instances or use cases where someone would want to put the backlink in the middle / top of the description?

Reply:
Yes: CIs workflow that parse the bottom of the message can break if you put them at the end.

Reply:
```suggestion
  - description: 'Added a new templating variable, batch_change_link, to give more control over where the "Created by Sourcegraph batch change ..." message appears in the published changeset description. This message would occasionally break CI workflows that parsed the bottom of a message; it can now be moved accordingly.'
```

How's this?